File: libs/ast_query/fixtures/sample.js
// Fixture for libs/ast_query JS tests. Known structure:
//   top-level function helper
//   top-level function greet (calls helper)
//   class Greeter with method speak (calls greet)
function helper(x) {
  return x + 1;
}

function greet(name) {
  return helper(name);
}

class Greeter {
  speak(name) {
    return greet(name);
  }
}

File: libs/ast_query/fixtures/sample.py
# Fixture for libs/ast_query tests. Known 1-based line numbers:
# helper=5, greet=9, Greeter=13, __init__=14, speak=17


def helper(x):
    return x + 1


def greet(name):
    return helper(name)


class Greeter:
    def __init__(self, name):
        self.name = name

    def speak(self):
        return greet(self.name)
